diff --git a/tunefs.ocfs2/op_query.c b/tunefs.ocfs2/op_query.c
index 295c185b..40394d15 100644
--- a/tunefs.ocfs2/op_query.c
+++ b/tunefs.ocfs2/op_query.c
@@ -339,7 +339,7 @@ static int query_run(struct tunefs_operation *op, ocfs2_filesys *fs,
 	register_printf_function('O', handle_ro_compat, handle_arginfo);
 
 	query_fs = fs;
-	fprintf(stdout, fmt);
+	fprintf(stdout, fmt, "dummy");
 	query_fs = NULL;
 
 	ocfs2_free(&fmt);
